mongodbatlas/resource_mongodbatlas_cluster_test.go

```diff
@@ -1553,7 +1553,6 @@ func testAccMongoDBAtlasClusterConfigTenant(projectID, name, instanceSize, diskS
 		provider_instance_size_name  = "%s"
 		//These must be the following values
  	 	mongo_db_major_version = "%s"
-		auto_scaling_disk_gb_enabled = false
```

Comment:
So what happens is this is false anyways?

Reply:
oh I got your point and you are right, if I change from true to false, the API will not receive the false

Let me change it

Reply:
oh but you cannot use TENANT cluster with `auto_scaling_disk_gb_enabled=true`

Reply:
so, we are ok (this condition only happens when the provider is TENANT)